src/test/ui/feature-gate/issue-43106-gating-of-builtin-attrs-error.rs

```diff
@@ -7,9 +7,6 @@
 
 // ignore-tidy-linelength
 
-// Crate-level is accepted, though it is almost certainly unused?
-#![inline]
```

Comment:
Did this attribute need to be removed? If this now produces as error then it should stay so the UI test can make sure it stays an error.